dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/DeepStorageBrowserPresenter.java

```diff
@@ -111,11 +112,22 @@
     @Inject
     private Workers workers;
 
+    @Inject
```

Comment:
Definitely going to go back and do a more pointed PR for the GuicePresenterInjector and GuiModule either next or soon